app/src/main/kotlin/com/wire/android/feature/conversation/content/usecase/MessageSendFailureHandler.kt

```diff
@@ -0,0 +1,26 @@
+package com.wire.android.feature.conversation.content.usecase
```

Comment:
I think this class shouldn't be here as it's not a usecase

Reply:
What about renaming the package to `domain`?

And inside we can have another package called `usecase` ?

Something like this:

```shell
|com
  |wire
    |android
      |feature
        |conversation
          |content
            |domain
              MessageSendFailureHandler.kt
              MessageSender.kt
              |usecase
                SendTextMessageUseCase.kt
                SendTextMessageUseCase.kt
```

Reply:
Yes it's better, we could also move data classes there also

Reply:
Done

Reply:
Maybe we should do the same for other packages for consistency